### PR TITLE
Feature/create contributing guide 177210106

### PR DESCRIPTION
**Pull request by Ajaxan**
Purpose
---------------
We want to have a guide that will allow any potential contributors to know what to do and what guidelines to follow.

Solution
---------------
Added the contributing.md file to give potential viewers/contributors a guide on how to add to the project.
NOTE: not all the links work. I wasn't sure where the links should go, might need to fix them in a future story.

Change summary
---------------
* Added contributing.ms file
* Sections on reporting and contributing and templates

Steps to Verify
----------------
1. Read Contributing file and make sure it all looks good.

### CODE REVIEW

**Review by shannonwells — APPROVED** [review]
lgtm, maybe you & @claireolmstead should both have project settings access if you're leading the charge on it.  @sbendar ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,124 @@
...
\ No newline at end of file
+# Contributing Guide
+
+## Welcome
+Welcome to the example client for demonstrating the use of the DSNP (Distributed Social Networking Protocol) SDK. Thank you for opening this file and considering contributing to the project either by reporting issues or directly contributing code to the project!
+
+## References
+In creating bug reports, feature requests, or pull requests it's important to be familiar and up to date with both: 
+* [DSNP Specs](https://www.google.com/search) for how the DSNP works
+* [DSNP SDK](https://www.google.com/search) for integration with the DSNP
```
Comment (wilwade) [inline CONTRIBUTING.md:9]:
```suggestion
* [DSNP Spec](https://github.com/LibertyDSNP/spec) for how the DSNP works
* [DSNP SDK](https://github.com/LibertyDSNP/sdk-ts) for integration with the DSNP
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+- A detailed descrtiption of what actually happened
+
+
+Additional details / screenshot
+---------------
+- Any suplmenental pictures or material
+- ![Screenshot]()
+```
+
+#### Feature Request Template
```
[the remark this answers is not in the document]
Comment (wilwade) [reply CONTRIBUTING.md:73]: You should be able to just add them to .github like https://github.com/LibertyDSNP/sdk-ts/tree/main/.github No additional access should be needed.

File: README.md [hunk 3]
```diff
@@ -20,6 +20,15 @@ __Testing:__
 * Tests written in [Jest](https://jestjs.io) and [Enzyme](https://enzymejs.github.io/enzyme/)
 * CI/CD done with [Github Actions](https://github.com/features/actions)
 
+## Project Setup
+Follow this quick start guide to get up and running. This guide assumes a working knowledge of npm and git. Make sure to check out the __CONTRIBUTING.md__ as well.
+* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
+* Install asdf: `asdf install`
```
Comment (wilwade) [inline README.md:26]: asdf is not a requirement, but just a common way to install the correct node version. Not sure how we want to communicate that however.

**Review by wilwade — CHANGES_REQUESTED** [review]
Two small things and a comment

**Comment by wilwade** [comment]
@Ajaxan you might want to check out https://github.com/LibertyDSNP/liberty-web/pull/13 as well to see the contributing guide over there and see what crossover we want.

**Comment by Ajaxan** [comment]
I added links to the liberty web contributing.md (or where it will be, once approved) as a baseline. I kept most of the specific contributing.md for the example client though as it gets more specific.

**Review by wilwade — APPROVED** [review]

**Comment by enddynayn** [comment]
👍

**Comment by jvasile** [comment]
Two things we might want to include in this doc:

 * Link to code of conduct (unless that info is in the contributing.md. The link to that was broken, so I'm not sure what the latest status is)
 * Link to forums and an encouragement to pre-flight work with us.  This gets people talking directly to us, which we want.  And it potentially avoids people doing work that we don't want to merge.

**Pull request merged**